src/containers/BookForm.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { createBook } from '../actions/index';
+
+class BookForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: null,
+      title: '',
+      category: '',
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleSubmit = event => {
+    const { createBook } = this.props;
+    event.preventDefault();
+    createBook(this.state);
+    event.target.reset();
```

Comment:
- [ ] OPTIONAL - Rather than resetting `event.target` simply clear state and the form fields would be empty. This would work however if you have beforehand implemented the previous suggestion.

Reply:
Done.